ivy/functional/frontends/jax/random.py

```diff
@@ -283,3 +285,29 @@ def pareto(key, b, shape=None, dtype="float64"):
     e = -ivy.log(1 - uniform)
 
     return ivy.exp(e / b)
+
+
+@handle_jax_dtype
+@to_ivy_arrays_and_back
+@with_supported_dtypes(
+    {
+        "0.4.13 and below": (
+            "float32",
+            "float64",
+        )
+    },
+    "jax",
+)
+def ball(key, d, p=2.0, shape=(), dtype="float64"):
+    seed = _get_seed(key)
+    d = operator.index(d)
+
+    g = ivy.gamma(1 / p, 1.0, shape=shape, dtype=dtype, seed=seed)
+    b = ivy.bernoulli(ivy.array([0.5]), shape=shape, dtype=dtype, seed=seed)
```

Comment:
There is shape inconsistency , which is throwing many exceptions, hard to trace for me, would you take your time and run the tests locally to inspect that🙂